frontend/src/pages/index.js

```diff
@@ -36,18 +35,22 @@ class LoginPage extends Component {
   }
 
   handleGoogle = async e => {
-    const result = await loginGoogleUser(e.tokenId)
-    const response = await result.json()
-    if (!response.success) {
+    const response = await loginGoogleUser(e.tokenId)
+    console.log(response)
+    if (response.status != 200) {
```

Comment:
nit: `success` not `status` (unless I'm missing some other context here)

Reply:
the auth server doesn't use the success boolean apparently, so when i changed it the first time it caused some errors